ocsp/responder.go

```diff
@@ -85,25 +85,44 @@ type Responder struct {
 // from an OCSP request to an OCSP response is done by the Source;
 // the Responder simply decodes the request, and passes back whatever
 // response is provided by the source.
+// Note: The caller must use http.StripPrefix to strip any path components
```

Comment:
is it possible that we drop GET and only go with POST? The code dealing with decoding base64 encoded URI makes me feel really hacky.